src/main/resources/application.yaml

```diff
@@ -1037,16 +1041,19 @@ security:
       - "uploadAdditionalEvidenceHomeOffice"
       - "uploadAddendumEvidenceHomeOffice"
       - "makeAnApplication"
+      - "decisionWithoutHearing"
```

Comment:
Does the `decisionWithoutHearing` event need to be added to the home office users too?
Would it be sufficient to just add it to case officer only?
(L1044, L1050, L1056)

Reply:
Removed the decisionWithoutHearing event from home office user role and test in my local successful